### PR TITLE
Added copyright message to docs

### PR DESCRIPTION
**Pull request by kolibril13**
Added a copyright message to the docs starting page and to the example page.
One thing to note regarding licence:
https://tldrlegal.com/license/mit-license
I think we want to use the MIT license, but on this page, I found, the sentence:
"You must include the copyright notice in all copies or substantial uses of the work."
But I think we do not want people to put in a copyright notice to everything they made with manim, do we?

### CODE REVIEW

**Comment by behackl** [comment]
> But I think we do not want people to put in a copyright notice to everything they made with manim, do we?

I am fairly certain that this is not what the MIT license requires, it only refers to our source code and **not** to content created using our library as a dependency.

In my understanding, the case where the license has to be included is when someone develops software which uses manim internally and thus contains an entire copy (or also just a "substantial part") of Manim's source code in their source code. Then our MIT license has to be shipped together with Manim's source code.

File: docs/source/index.rst [hunk 1]
```diff
@@ -15,6 +15,9 @@ This project is still very much a work in progress, but I hope that the
 information here will make it easier for newcomers to get started using
 ``Manim``.
 
+.. tip:: All content of the docs are under MIT license. Especially for the examples you encounter:
+Feel free to use this code in your own projects!
+
```
Comment (behackl) [inline docs/source/index.rst:20]: Directives (`::`) should usually be written with a newline after `::`, followed by an indented block.

```suggestion
.. tip:: 

    All content of the docs is licensed under the MIT license. Especially for the examples
    you encounter: Feel free to use this code in your own projects!

```

**Review by behackl — APPROVED** [review]
LGTM in general, the tip block has to be fixed though.

**Comment by kolibril13** [comment]
> In my understanding, the case where the license has to be included is when someone develops software which uses manim internally and thus contains an entire copy (or also just a "substantial part") of Manim's source code in their source code. Then our MIT license has to be shipped together with Manim's source code.

I just looked at some softwareengineering.stackexchange.com answers and found similar understandings. 
Then I think this is the right licence for us to use!

**Review by leotrs — APPROVED** [review]

**Pull request merged**